backend/src/models/review.js

```diff
@@ -20,14 +20,24 @@ class Review {
      * @param   {number}  pageNumber    The page number for which we want to get questions.
      * @returns {Array}
      */
-    getReviews(code, pageNumber) {
-        const pageSize = 10
+    getReviews(code, pageNumber, pageSize) {
         const offset = (pageSize * pageNumber) - pageSize
         return this.db
             .queryAll('SELECT * FROM review WHERE code=? ORDER BY timestamp DESC LIMIT ?, ?',
                 [code, offset, pageSize])
     }
 
+    /**
+     * Gets the total number of reviews for a course
+     * @param   {string} code        The code of the course
+     * @returns {object}
+     */
+    getReviewCount(code) {
```

Comment:
It would be maybe more understandable if this was separated if you only made the call on the initial getQuestions/getReviews call. But as we're returning with every request (which is also fine), this should be a subquery. Eg from databases.

```sql
select c.year, c.term, count(*), c.starting
from courses c
group by c.year, c.term, c.starting
order by c.starting
```

Reply:
But maybe not possible in lite.

Reply:
I tried doing something like this 
```
'SELECT q.title, q.body, COUNT(*) FROM question q WHERE code=? GROUP BY q.title, q.body ORDER BY timestamp DESC LIMIT ?, ?'
```
However COUNT is always 1, also It doesn't seem to work when using * to select all fields of the question. Might look at it later but the current solution works and is only slightly less efficient.